app/settings/settings.routes.js

```diff
@@ -71,6 +71,14 @@ function (
             template: require('./surveys/survey-edit.html')
         }
     )
+    .state(
+        {
+            name: 'settings.surveys.targeted',
+            url: '/create-targeted-survey',
```

Comment:
Can this be /targetedSurvey/createinstead? to follow the /:thing/:action way that we are using for other routes?

Reply:
For sure. I hadn't even noticed the pattern. Looks like I should open my eyes more :)